jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/PortForwardServicePortOrderTest.java

```diff
@@ -49,10 +50,11 @@ void portsSpecifiedInCorrectOrderPortForward(int containerPort, int localPort) t
         request.complete(r);
         return "";
       }).always();
-    PortForwardService.forwardPortAsync(kubernetesClient, "foo-pod", containerPort, localPort);
-    // When
-    try (final Socket ignored = new Socket(InetAddress.getLocalHost(), localPort)) {
-      // The socket connection triggers the KubernetesClient request to the k8s portforward endpoint
+    try (LocalPortForward localPortForward = PortForwardService.forwardPortAsync(kubernetesClient, "foo-pod", containerPort, localPort)) {
+      // When
+      try (final Socket ignored = new Socket(InetAddress.getLocalHost(), localPort)) {
+        // The socket connection triggers the KubernetesClient request to the k8s portforward endpoint
+      }
     }
```

Comment:
Looks good, but could we merge this into a single try-finally block?
```suggestion
    try (
      LocalPortForward localPortForward = PortForwardService.forwardPortAsync(kubernetesClient, "foo-pod", containerPort, localPort);
      // When
      final Socket ignored = new Socket(InetAddress.getLocalHost(), localPort)
    ) {

      // The socket connection triggers the KubernetesClient request to the k8s portforward endpoint
    }
```